lab-sarah/lib/parse-json.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = function(req){
+  return new Promise((resolve, reject) => {
+    if (req.method === 'POST' || req.method === 'PUT'){
+      console.log('parsing body');
```

Comment:
Please remove console.logs that are unnecessary prior to submission - this includes nearly everything except logging errors to the server as well as messages like "server is up on such-and-such port."